### DISCUSSION TITLE
Page loading indicator (throbber)

### DISCUSSION THREAD

**Discussion by Someone-G** [Aris-t2/CustomCSSforFx#493] [Q&A]
Hello. Tell me how you can use only the page loading indicator (throbber)? I don't want to install CustomCSSforFx completely. The browser is clean. Thanks.

**Comment by Aris-t2** [comment]
CSS can not restore the throbber / page loading indicator.

There is a JavaScript project, that offers a script to achieve this: https://github.com/Aris-t2/CustomJSforFx

(instructions how to use custom JS scripts are on main page)

Script: https://github.com/Aris-t2/CustomJSforFx/blob/master/scripts/activity_throbber.uc.js
